frontend/src/sass/_index.scss

```diff
@@ -0,0 +1,3 @@
+@forward 'color-palettes';
```

Comment:
Feel free to remove the files that aren't needed.

Reply:
I removed `_index.scss`